### PR TITLE
Add support for creation/deletion of kafka

### PR DESCRIPTION
**Pull request by yash-nisar**
Closes https://github.com/Azure/azure-cli-extensions/issues/6330

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yash-nisar** [comment]
@zhoxing-ms Can you take a look at the PR ?

**Comment by zhoxing-ms** [comment]
@yash-nisar Could you please take a look at this comment https://github.com/Azure/azure-cli-extensions/pull/6345#issuecomment-1568901221 first?

File: src/containerapp/azext_containerapp/_utils.py [hunk 1]
```diff
@@ -453,7 +453,7 @@ def process_service(cmd, resource_list, service_name, arg_dict, subscription_id,
                     raise ResourceNotFoundError(f"The service '{service_name}' does not exist")
 
                 configuration = containerapp_def["properties"]["configuration"]["service"]
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/_utils.py:455]: Recomment to use safe_get:

```
service_type = safe_get(containerapp_def, "properties", "configuration", "service", "type"):
if service_type is None or service_type not in DEV_SERVICE_LIST:
```

**Comment by Greedygre** [comment]
Please write the description of changes into HISTORY.rst.

File: src/containerapp/azext_containerapp/_utils.py (hunk 1 above)
Comment (yash-nisar) [reply src/containerapp/azext_containerapp/_utils.py]: Done

**Comment by yash-nisar** [comment]
> Please write the description of changes into HISTORY.rst.

Done

**Review by Greedygre — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by Moazzem-Hossain-pixel — COMMENTED** [review]
Thanks

**Pull request merged**